common/src/main/scala/explore/AppMain.scala

```diff
@@ -101,11 +105,30 @@ trait AppMain extends IOApp {
       IO.fromFuture(httpCall).map(_.body)
     }
 
+    val fetchToken: IO[String] = IO("...")
+
+    val connectParameters: IO[JsonObject]                          =
+      fetchToken.map(token => JsonObject("Authorization" -> s"Bearer $token".asJson))
+
+    val reconnectionStrategy: ReconnectionStrategy[IO, CloseEvent] =
+      ReconnectionStrategy(maxAttempts = 100,
+                           (attempt, event) =>
+                             if (event.code === 1000)
+                               none
+                             else
+                               FiniteDuration(5L * attempt, TimeUnit.SECONDS).some
+      )
+
     for {
       _         <- logger.info(s"Git Commit: [${BuildInfo.gitHeadCommit.getOrElse("NONE")}]")
       appConfig <- fetchConfig
       _         <- logger.info(s"Config: ${appConfig.show}")
       ctx       <- AppContext.from[IO](appConfig)
+      _         <- ctx.clients.odb
+                     .connect(
+                       connectParameters,
+                       reconnectionStrategy.asInstanceOf[ReconnectionStrategy[IO, ctx.clients.odb.CE]].some
```

Comment:
I'm working of ways to improve the types in `clue` so that this cast is not necessary. I will investigate using `sttp` but it will take a while, so I'm pushing this in the meantime not to hold things up.